tests/testthat/test_OneStageDesign.R

```diff
@@ -115,22 +115,24 @@ test_that("OneStageDesign can be converted to TwoStageDesign", {
     design2 <- TwoStageDesign(design1, order = 7)
     pow     <- Power(Normal(two_armed = FALSE), PointMassPrior(.3, 1))
 
-    expect_true(
-        length(design2@x1_norm_pivots) == 7)
-
-    expect_true(
-        length(design2@n2_pivots) == 7)
+    expect_true(length(design2@x1_norm_pivots) == 7)
+    expect_true(length(design2@n2_pivots) == 7)
 
     expect_equal(
         evaluate(pow, design1),
         evaluate(pow, design2),
         tolerance = 1e-6, scale = 1)
 
     expect_equal(
-        evaluate(pow, design1),
+        evaluate(pow, design1, optimization = TRUE),
         .8,
         tolerance = 1e-5, scale = 1)
 
+    expect_lte(
+        evaluate(pow, design1, optimization = FALSE),
+        .8)
```

Comment:
i don't get the rationale for that test - shouldn't this test for closeness (with a certain tolerance) to .8?